test/SlowTests/Client/UniqueValues.cs

```diff
@@ -295,5 +295,55 @@ public async Task CompareExchangeShouldBeRemovedFromStorageWhenDbGetsDeleted()
                 Assert.Equal(0, resultItems);
             }
         }
+
+        [Fact]
+        public async Task EnsureCanCompareExchangeWithCorrectCharsEscapeInKey()
+        {
+            DoNotReuseServer();
+            var store = GetDocumentStore();
+
+            const string cmpXchgKey1 = "emails/foo+test@email.com";
```

Comment:
can't we wrap this with a method and call it with arg and expected result to avoid code duplication?

Reply:
ofc I can, but I am not sure if this is necessary for test.